src/platforms/android/manual-configuration.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: Manual Configuration
+sidebar_order: 2
+description: "Learn how to manually configure the Sentry Android SDK."
+---
+<PlatformContent includePath="getting-started-primer" />
+
+<PlatformSection noGuides notSupported={["android", "dart", "elixir", "flutter", "perl", "react-native", "unity", "unreal"]}>
+
+<Alert level="info" title="Using a framework?">
+
+Get started using a guide listed in the right sidebar.
+
+</Alert>
+
+</PlatformSection>
+
+## Installation
+
+If you don't want the Sentry Gradle plugin to install the Sentry SDK automatically, you can define the dependency directly in the `build.gradle` file of your app module.
+
+```groovy {filename:app/build.gradle}
+// Make sure mavenCentral is there.
+repositories {
+    mavenCentral()
+}
+
+// Enable Java 1.8 source compatibility if you haven't yet.
+android {
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+    }
+}
+
+// Add Sentry's SDK as a dependency.
+dependencies {
+    implementation 'io.sentry:sentry-android:{{ packages.version('sentry.java.android', '4.2.0') }}'
+}
+```
+
+```kotlin {filename:app/build.gradle.kts}
+// Make sure mavenCentral is there.
+repositories {
+    mavenCentral()
+}
+
+// Enable Java 1.8 source compatibility if you haven't yet.
+android {
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+    }
+}
+
+// Add Sentry's SDK as a dependency.
+dependencies {
+    implementation("io.sentry:sentry-android:{{ packages.version('sentry.java.android', '4.2.0') }}")
+}
+```
+
+<Note>
+
+The [NDK integration](/platforms/android/using-ndk/) comes packaged with the SDK and requires API level 16, though other levels are supported. If you're using multiple Sentry dependencies, you can add a [bill of materials](/platforms/android/configuration/bill-of-materials) to avoid specifying the version of each dependency.
+
+</Note>
+
+## Configuration
+
+<PlatformContent includePath="getting-started-config" />
+
+<Note>
+
+Don't already have an account and Sentry project established? Head over to [sentry.io](https://sentry.io/signup/), then return to this page.
+
+</Note>
+
+## Verify
+
+<PlatformSection notSupported={["unreal"]}>
+
+This snippet includes an intentional error, so you can test to make sure that everything's working as soon as you set it up:
+
+</PlatformSection>
+
+<PlatformContent includePath="getting-started-verify" />
+
+To view and resolve the recorded error, log into [sentry.io](https://sentry.io) and open your project. Clicking on the error's title will open a page where you can see detailed information and mark it as resolved.
+
+<Note>
+
+Learn more about manually capturing an error or message in our <PlatformLink to="/usage/">Usage documentation</PlatformLink>.
+
+</Note>
+
```

Comment:
I guess we can remove everything below that, since it's an android-specific page? 

Also, not sure if it's intended, but the "Next Steps" section is not displayed here actually:
![image](https://user-images.githubusercontent.com/4999776/218737774-414ddc65-15a5-441b-97c4-6b00b80052a6.png)